src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
+
+    List<Driver> getAll();
+
+    Driver update(Driver manufacturer);
```

Comment:
The parameter name 'manufacturer' should be 'driver' to accurately represent the object 🚗